framework/src/ble_manager/ble_manager_api.c

```diff
@@ -70,6 +70,13 @@ ble_result_e ble_manager_set_secure_param(ble_sec_param *sec_param )
 	RETURN_RESULT(res, msg);
 }
 
+ble_result_e ble_manager_start_bond(ble_conn_handle *conn_handle)
```

Comment:
```suggestion
ble_result_e ble_manager_start_bond(ble_conn_handle conn_handle)
```

Is there a reason to pass the conn_handle value as a pointer? If there's no significant reason, it seems better to just receive the value.

Reply:
its modified as suggested